spec/pdc/v1/release_spec.rb

```diff
@@ -23,6 +23,11 @@
       count = release.where(active: false).count
       count.must_equal 0
     end
+
+    it 'can get release by multi product_version ' do
```

Comment:
isn't this test missing the vcr files? i.e. when you run the test, doesn't it create VCR files?

Reply:
Yes, it creates a yml file.
Do you mean this can_get_release_by_multi_product_version_.yml? It is automatically generated.
And one place need to change, shouldn't add the blank at the end of the name. After the change the name should be can_get_release_by_multi_product_version.yml, please help to review.

Reply:
@xiangge those files need to be checked in just like the others, otherwise the test results can vary depending on the data in the server which isn't a behaviour we want.